docs/mediapipe.md

```diff
@@ -9,120 +9,63 @@
+* <a href="#ovms-calculators">OpenVINO Model Server Calculators</a>
+* <a href="#create-graph">How to create the graph for deployment in OpenVINO Model Server</a>
+* <a href="#graph-deploy">Graph deployment</a>
+* <a href="#testing">Deployment testing</a>
+* <a href="#client">Using MediaPipe graphs from the remote client </a>
+* <a href="#updating-graph">How to update existing graphs to use OV for inference </a>
+* <a href="#adding-calculator">Adding your own MediaPipe calculator to OpenVINO Model Server </a>
+* <a href="#graph-examples">Demos and examples</a>
+* <a href="#current-limitations">Current Limitations</a>
 
-## Graph proto files <a name="graph-proto"></a>
 
-Graph proto files are used to define a graph. Example content of proto file with graph containing OpenVINO inference nodes:
 
-```
+## OpenVINO calculators <a name="ovms-calculators"></a>
 
-input_stream: "in1"
-input_stream: "in2"
-output_stream: "out"
-node {
-  calculator: "OpenVINOModelServerSessionCalculator"
-  output_side_packet: "SESSION:dummy"
-  node_options: {
-    [type.googleapis.com / mediapipe.OpenVINOModelServerSessionCalculatorOptions]: {
-      servable_name: "dummy"
-      servable_version: "1"
-    }
-  }
-}
-node {
-  calculator: "OpenVINOModelServerSessionCalculator"
-  output_side_packet: "SESSION:add"
-  node_options: {
-    [type.googleapis.com / mediapipe.OpenVINOModelServerSessionCalculatorOptions]: {
-      servable_name: "add"
-      servable_version: "1"
-    }
-  }
-}
-node {
-  calculator: "OpenVINOInferenceCalculator"
-  input_side_packet: "SESSION:dummy"
-  input_stream: "DUMMY_IN:in1"
-  output_stream: "DUMMY_OUT:dummy_output"
-  node_options: {
-    [type.googleapis.com / mediapipe.OpenVINOInferenceCalculatorOptions]: {
-        tag_to_input_tensor_names {
-          key: "DUMMY_IN"
-          value: "b"
-        }
-        tag_to_output_tensor_names {
-          key: "DUMMY_OUT"
-          value: "a"
-        }
-    }
-  }
-}
-node {
-  calculator: "OpenVINOInferenceCalculator"
-  input_side_packet: "SESSION:add"
-  input_stream: "ADD_INPUT1:dummy_output"
-  input_stream: "ADD_INPUT2:in2"
-  output_stream: "SUM:out"
-  node_options: {
-    [type.googleapis.com / mediapipe.OpenVINOInferenceCalculatorOptions]: {
-        tag_to_input_tensor_names {
-          key: "ADD_INPUT1"
-          value: "input1"
-        }
-        tag_to_input_tensor_names {
-          key: "ADD_INPUT2"
-          value: "input2"
-        }
-        tag_to_output_tensor_names {
-          key: "SUM"
-          value: "sum"
-        }
-    }
-  }
-}
+We are introducing a set of calculators which can bring to the graphs execution the advantage of OpenVINO Runtime.
+
+Check their [documentation](https://github.com/openvinotoolkit/mediapipe/blob/main/mediapipe/calculators/ovms)
 
-```
 
-Here can be found more information about [MediaPipe graphs proto](https://developers.google.com/mediapipe/framework/framework_concepts/graphs)
+## Python calculator
```

Comment:
It will be quite unhandy to use our PythonExecutorCalculator outside OVMS as of now.